src/mudlet.pro

```diff
@@ -284,7 +284,7 @@ unix:!macx {
         LIBS +=  \
             -L"$${MINGW_BASE_DIR_TEST}\\bin" \
             -llua51 \
-            -llibhunspell-1.6
+            -lhunspell-1.6
```

Comment:
How does this fix things? If you really want to do without the `lib` prefix you need to also do a `CONFIG += no_plugin_name_prefix` according to this Stack Overflow Answer: https://stackoverflow.com/a/54696182/4805858 ...

Reply:
qmake is broken in 5.14 and it doesn't work for that specific version... works in all others. While the no `lib` version works in all of them.

Reply:
Even ***without*** that `CONFIG` option?

:confounded:

Reply:
Could you approve the change?